environments.toml

```diff
@@ -0,0 +1,1409 @@
+cve = ["CVE-2022-22965"]
+app = "Spring"
+path = "spring/CVE-2022-22965"
+
+[[environment]]
+name = "Spring Security Authorization Bypass in RegexRequestMatcher"
+cve = ["CVE-2022-22978"]
+app = "Spring"
+path = "spring/CVE-2022-22978"
+
+[[environment]]
+name = "S2-001 Remote Code Execution"
+cve = [""]
+app = "Struts2"
+path = "struts2/s2-001"
+
+[[environment]]
+name = "S2-005 Remote Code Execution"
+cve = ["CVE-2010-1870"]
+app = "Struts2"
+path = "struts2/s2-005"
+
+[[environment]]
+name = "S2-007 Remote Code Execution"
+cve = [""]
+app = "Struts2"
+path = "struts2/s2-007"
+
+[[environment]]
+name = "S2-008 Remote Code Execution"
+cve = ["CVE-2012-0391"]
+app = "Struts2"
+path = "struts2/s2-008"
+
+[[environment]]
+name = "S2-009 Remote Code Execution"
+cve = ["CVE-2011-3923"]
+app = "Struts2"
+path = "struts2/s2-009"
+
+[[environment]]
+name = "S2-012 Remote Code Execution"
+cve = ["CVE-2013-1965"]
+app = "Struts2"
+path = "struts2/s2-012"
+
+[[environment]]
+name = "S2-013 Remote Code Execution"
+cve = ["CVE-2013-1966"]
+app = "Struts2"
+path = "struts2/s2-013"
+
+[[environment]]
+name = "S2-015 Remote Code Execution"
+cve = ["CVE-2013-2134", "CVE-2013-2135"]
+app = "Struts2"
+path = "struts2/s2-015"
+
+[[environment]]
+name = "S2-016 Remote Code Execution"
+cve = ["CVE-2013-2251"]
+app = "Struts2"
+path = "struts2/s2-016"
+
+[[environment]]
+name = "S2-032 Remote Code Execution"
+cve = ["CVE-2016-3081"]
+app = "Struts2"
+path = "struts2/s2-032"
+
+[[environment]]
+name = "S2-045 Remote Code Execution"
+cve = ["CVE-2017-5638"]
+app = "Struts2"
+path = "struts2/s2-045"
+
+[[environment]]
+name = "S2-046 Remote Code Execution"
+cve = ["CVE-2017-5638"]
+app = "Struts2"
+path = "struts2/s2-046"
+
+[[environment]]
+name = "S2-048 Remote Code Execution"
+cve = ["CVE-2017-9791"]
+app = "Struts2"
+path = "struts2/s2-048"
+
+[[environment]]
+name = "S2-052 Remote Code Execution"
+cve = ["CVE-2017-9805"]
+app = "Struts2"
+path = "struts2/s2-052"
+
+[[environment]]
+name = "S2-053 Remote Code Execution"
+cve = ["CVE-2017-12611"]
+app = "Struts2"
+path = "struts2/s2-053"
+
+[[environment]]
+name = "Struts2 S2-057 Remote Command Execution"
+cve = ["CVE-2018-11776"]
+app = "Struts2"
+path = "struts2/s2-057"
+
+[[environment]]
+name = "Struts2 S2-059 Remote Command Execution"
+cve = ["CVE-2019-0230"]
+app = "Struts2"
+path = "struts2/s2-059"
+
+[[environment]]
+name = "Struts2 S2-061 Remote Command Execution"
+cve = ["CVE-2020-17530"]
+app = "Struts2"
+path = "struts2/s2-061"
+
+[[environment]]
+name = "Supervisord Remote Command Execution"
+cve = ["CVE-2017-11610"]
+app = "Supervisor"
+path = "supervisor/CVE-2017-11610"
+
+[[environment]]
+name = "ThinkPHP 2.x Arbitrary Code Execution"
+cve = [""]
+app = "ThinkPHP"
+path = "thinkphp/2-rce"
+
+[[environment]]
+name = "ThinkPHP5 5.0.22/5.1.29 Remote Code Execution"
+cve = [""]
+app = "ThinkPHP"
+path = "thinkphp/5-rce"
+
+[[environment]]
+name = "ThinkPHP5 5.0.23 Remote Code Execution"
+cve = [""]
```

Comment:
just use `""` or `[]` to represent zero value, not `[""]`.

Reply:
@phith0n fixed.